controller/projects.js

```diff
@@ -3,7 +3,7 @@ const fs = require('fs');
 const _ = require('lodash');
 const mkdirp = require('mkdirp');
 const rimraf = require('rimraf');
-const fsJson = require('fs-json')();
+const fsJson = require('../vendor/fs-json')();

```

Comment:
I added a note to the file, but this module was created by John if we want to maintain it https://www.npmjs.com/package/fs-json